SConstruct

```diff
@@ -2490,6 +2502,10 @@ def doConfigure(myenv):
         # http://lists.llvm.org/pipermail/cfe-dev/2015-November/045852.html
         #
         if env.TargetOSIs('posix') and not env.ToolchainIs('clang') and conf.CheckForFortify():
+            if conf.CheckForFortifyDefined():
+                conf.env.Prepend(
+                    CPPFLAGS='-U_FORTIFY_SOURCE'
```

Comment:
That's what I did initially, but I think the default predefined value for _FORTIFY_SOURCE is 1 and this is setting it to 2, which enables more checking.

From http://man7.org/linux/man-pages/man7/feature_test_macros.7.html it says:

> If _FORTIFY_SOURCE is set to 1, with compiler optimization level 1 (gcc -O1) and above, checks that shouldn't change the behavior of conforming programs are performed.  With _FORTIFY_SOURCE set to 2, some more checking is added, but some conforming programs might fail.